features/support/ui/landing.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'ui'
+
+module UI
+  class Landing < SitePrism::Page
+    set_url '/'
+    set_url_matcher(/wpcc\/\d+\/landing/)
+
+    element :age, '#your_details_form_age'
```

Comment:
@Tr4pSt3R @margOnline @willhall88 we need to agree together on this t- (read Jones comments above). 

BUT ... for this elements, are they inputs right? I would change to:


```
element :age, input[name='your_details_form[age]']
```

And the other as well and then I am happy.